src/test/fuzz/tx_pool.cpp

```diff
@@ -101,9 +101,8 @@ void Finish(FuzzedDataProvider& fuzzed_data_provider, MockedTxPool& tx_pool, Cha
     if (!info_all.empty()) {
         const auto& tx_to_remove = *PickValue(fuzzed_data_provider, info_all).tx;
         WITH_LOCK(tx_pool.cs, tx_pool.removeRecursive(tx_to_remove, MemPoolRemovalReason::BLOCK /* dummy */));
-        std::vector<uint256> all_txids;
-        tx_pool.queryHashes(all_txids);
-        assert(all_txids.size() < info_all.size());
+        const auto new_size{WITH_LOCK(tx_pool.cs, return tx_pool.entryAll().size())};
```

Comment:
could use `infoAll` to avoid the `WITH_LOCK`?

Reply:
Looks like there's a much simpler `tx_pool.size()` option too, using that now.